operator/pkg/resources/configuration.go

```diff
@@ -48,6 +48,34 @@ func CreateConfiguration(
 
 	mountPoints := resourcetypes.GetTLSMountPoints()
 
+	addresses, err := generateSeedServer(ctx, pandaCluster, serviceFQDN, client)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	cfg.Node.Rpk.KafkaAPI.Brokers = []string{}
+	cfg.Node.Rpk.KafkaAPI.Brokers = append(cfg.Node.Rpk.KafkaAPI.Brokers, addresses...)
+	if tls := tlsConfigProvider.KafkaClientBrokerTLS(mountPoints); tls != nil {
```

Comment:
What was the `KafkaClientBrokerTLS` helper used for previously? I guess I understand why you needed to implement the Admin API and Schema Registry API helpers and configuration, but it seems odd that this part wasn't here when the helper already existed... was that intentional?

Reply:
Also, just noticed the double line initialization of `cfg.Node.Rpk.KafkaAPI.Brokers` with an `append`, think you just want:

```
	cfg.Node.Rpk.KafkaAPI.Brokers = addresses
```